### ISSUE TITLE
Deploy to maven central

### ISSUE THREAD

**Issue by mojo2012** [shannah/Java-Objective-C-Bridge#17]
We should deploy the lib to maven-central. I have an account and can setup a proper build toolchain, if you want

**Comment by shannah** [comment]
I have an older version deployed to maven central.  

~~~~
<dependency>
  <groupId>ca.weblite</groupId>
  <artifactId>java-objc-bridge</artifactId>
  <version>1.0.0</version>
</dependency>
~~~~

I should be able to add your sonatype username to the allowed users so that you can publish to this yet.  (Don't know how to do this yet, probably need to file an issue with Sonatype).

**Comment by mojo2012** [comment]
Yes you have to file an issue and hope someones ever gonna look at it :-D Did you create this artifact manually or how did you do it in the first place?

**Comment by shannah** [comment]
I had it built into the pom file.  The mvn release (if I recall) automatically submitted to the staging repo.  Then I had to log into their web ui, and close the staging repo, then release it.

The maven branch is still there, but stale.
https://github.com/shannah/Java-Objective-C-Bridge/tree/maven

I never actually used it (the maven build) in a real project, so it was more of an experiment than anything.

**Comment by mojo2012** [comment]
Could you have a look at the maven build and merge the PR? I think we should be good to go to push to nexus
